README.md

```diff
@@ -169,6 +176,8 @@ var s3Options = {
   baseUrl: process.env.SPACES_BASE_URL, 
   region: process.env.SPACES_REGION,
   directAccess: true,
+  preserveFilename: "always",
+  fileNameCheck: "safe",
```

Comment:
FWIW: I'm only using these settings in my app.   Happy to remove the others if they are at controversial.